src/components/Image.web.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import {
+  Image as NativeImage,
+  StyleProp,
+  ImageStyle,
+  ImageProps,
+  LayoutChangeEvent,
+} from "react-native";
+import AspectRatio from "./AspectRatio.web";
+import Config from "./Config";
+
+interface Props extends ImageProps {
+  source: any;
+  style?: StyleProp<ImageStyle>;
+  resizeMode?:
```

Comment:
do you think there's a type definition lib that supports these? or is it possible to create an enum with a shared file for these? like `enum ResizeMode =`?